From 7196c9441af941be74180f8e8d4e10a08659b80f Mon Sep 17 00:00:00 2001
From: Jesse Barnes <jbarnes@virtuousgeek.org>
Date: Thu, 5 May 2011 09:21:19 -0700
Subject: [PATCH 1/2] Revert "glxproto: make GLX swap event struct match spec"

This reverts commit f5403828c68af0f12c79c0504df0a4781ca84b2b.

This broke compatibility with existing code, so revert in favor of a new
struct definition instead.
---
 glxproto.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/glxproto.h b/glxproto.h
index dfa0647..0ff44e3 100644
--- a/glxproto.h
+++ b/glxproto.h
@@ -1375,13 +1375,13 @@ typedef struct {
     BYTE pad;
     CARD16 sequenceNumber B16;
     CARD16 event_type B16;
-    CARD16 pad2;
     CARD32 drawable;
     CARD32 ust_hi B32;
     CARD32 ust_lo B32;
     CARD32 msc_hi B32;
     CARD32 msc_lo B32;
-    CARD32 sbc B32;
+    CARD32 sbc_hi B32;
+    CARD32 sbc_lo B32;
 } xGLXBufferSwapComplete;
 
 /************************************************************************/
-- 
1.7.1

